bundle/manifests/keda-olm-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,344 @@
tTwnTujG9qZWl8+pL1Vje2+bcVuNpjDqmG2e5X32jmp53n3fnX1tq/z56AbjEIZNFU8wrrQdW8nQubqa85fr83F5XeLo0idSWeCRYZgFAADAhGpmyOMiGxvidzPd1zXqjDGtZntxNDZlLaNGbRIxa0v2trU8Pkj6yku7p6KkQUySq7Wjs7KZ6ZTGZXWYq/x85kI3H46reOQELw7wCHGbMQAAAKZAU8uzo7OZqssqI5oZk4JNEu1CN2sv77t9Lc9bP6v+4cfvVb8UM67iaVJZG0xZia/nNDlXd/2gvRzW1lfiaJJ+/jipLPAIkcwCAABgoo1reT7QMKfqTWkyZ4yVZptxtCrWPLfWetboZ6+wqFP9pa9f/baI5Nr0yqoMU9Mrm1wlrawObs4vtKuwWMk4lT3OEAs8agyzAAAAmHA3anlia06bWp4wSmUzk1K0h5/Mnrnd+r7zTvnGLy+lbbUyaFLZ2AyyRsu60616puPK7GLY9mXsrTZVPJKaSllSWeDRYpsxAAAAJldTyyNn1D6zpPNB1VeVqdWYTMVIjFZUzeqSfeqz1rcziIMvff3qG5JiHpLkRnWYJS28mioV3sWW+oX2Qqjztbh5+p0kJ5shVm+6ShnAo0AyCwAAgMn0G7U8dZ4brzqu5fFiko5+1zVP7bdLn7W+r785/FZexitNIitBhs2lTz75Sn1WJ+24ou1Cf3Et9M42qeyx0U9NzWlZAI8UwywAAAAmzq1reWoT1RuTok2iJiXRJ/aZfbda34eXwsd//MrVH0pKuZrxWdlgTGnEVKLe1T3n+5eK0LmwlTY3NiJVPMDjg23GAAAAmEB78+SGiD9/Xpd6V7Ro97QdZjV53zTzqEZv1Fg9tJS1b7W+v/i7wYspmd0kOpAYhlZMYbwpdcbWdTX081eeCll+MG7lh6Os8p4AjxOSWQAAAEy4ppbHaHtYmFA1yayapME08e3+edM60DOfGuD8/AN39mtv5j8Z1fBEPxSRYTRaasvXMnSurub85fp8XF6XOLr0iVQWeKwwzAIAAGBi3ajlMRpnVGMmmqJoMs0NTVb+3ZHsC/optTwxSvrqqd0XRudkr1XxtKRothcbl9VhrvLzmQvdfDiu4pETvCTAY4ZtxgAAAJhgTS3PRQ02qto5jaE2ZkbVNgOtiBw6kH3x09b24/frt35wpvoXVTNQk/Ik2TDzqayir+c0OVd3/aC9HNbWV+JoT/Pzx0llgccMySwAAAAm0+mTWu6cGeWusTYaW6W2rI6S2fF6gqwuZb910rV2yf2Xr+2+OL69OAzFyNB6X7rm0qdWVgdX+YV2FRYrGaeyxxligccRwywAAAAmz2i8PDaq5XELRhcWRJIfbyjOoh99VzHxiX1m+TfX9tbPqld//qH7SJPkam3eVPGE5qysd3UytXMLXb/ty9hbbap4JDWjMaks8PhhmAUAAMBEuVbLc0xEmlqeMLysoVaNYZzKepOlEFOKkuLyoj1w89oGRer/x7/pvywm5b45J5tCEYIbVfGYVlbP+znfDS4uVlfj5unt0aVPIp9y6BbAI8cwCwAAgAkzDkm3tzf3pswmfF2Q2euL8KLWxiQSVg7Yzs1r+9t3i7/+pB93JOgwCzJM0ioy0ylNq1V7X/viigv9xbXw7M7RKOvHRj8oNadlATx2GGYBAAAwgY6POmavCW6gRZPaGkkm2ZhMCs8cNNn++Ru1PJ/sxg//08n+t5PKcHR7cdZsL/ZlFW5U8fRtEToXttJJqniAxx7DLAAAACbYEfHlJ2pb88nYTnI2JZMkmpDCv39u9vDN6zr1dvGnrpZcogyTabYYa2F079Knm6p4tppUlioe4LFHNQ8AAAAmWtY5kET6olknaYgxmpkg0fkn9pt/e21dH+2EH//hi/23RLXKRIoUfCmaquilTi11rqKKB5g0JLMAAACYYOdGHz2fScm4YWrZmRCz6EWNe2LRPiMyvon4/7wx/KOmgudaIhuMjKp4THt86RNVPMDkYZgFAADARLNzMZkypiyG6OsQWjF6tbY+uGhWZJzKnvrL7w5PNxU82WiQNWUrmyulndUhVN4vVVTxABOIYRYAAAAT6IQsn95I7aUjqdWPaSb0oredEEwMajKnIdT7582TIUr9J6f6X0nRFU0iG70prfqqCL5Oece5uusvn6+o4gEmEMMsAAAAJtLJ5kOfPiMz3ZAGPsRO5kM7OC+VdybY+tBSNvOrnfA/XtkKv4itduG9KWtxVSW2SqZ2dc/5q+3qxqVPVPEAE0X5vwoAAIBJoqo66po9dtIc3f+suVwv2YWDs7bVn8lEypbJsqyydWajsamlqi4lk0mKoeVD8zXrfOVLv1h1fbm64pdF4qZIpIoHmCwkswAAAJgo15PT9WOpd3Y3dQ8PYze4OPAfx6ZiR7V24rI6S1llpaqa700aa9quTlo53+q4ZpC92s7DaJA9fXJvezGAScIwCwAAgAl1QjaXN1LnQpGaC5zms4XQnIENrvLN0DpsV3Xusrr5rnVZy27hos66+SsuNIPs+JyspBvbixlogUnCNmMAAABMnOtbjZ8Xlc1Nc/T3NvRq+5xZzjomv9QyoWfUz+v1m5yyQUod7+Pl1ZWYXT6XmkF2fE5WEtuLgcnEMAsAAICJpNpcO3xC5dhzKtvLevT3elquHtX8/Hkjclj8/g9uDLM7PmU9n9of1Wl5/bm9RFauby9mmAUmD8MsAAAAJtZooH3+hIocFzkturEuunthS0WOSrlzZjTMdvY/N/qFt7cqaXNzU2R5e7y1mEEWmGgMswAAAJhY17cbN5otx6dPqsixWyznpIzPx54Qef74qE+WGh5gcjHMAgAAYOKNtxzvuZbU/pprA+wYaSww+RhmAQAAMDV+baj9FAyxwPRgmAUAAAAATBx6ZgEAAAAAE4dhFgAAAAAwcRhmAQAAAAATh2EWAAAAADBxGGYBAAAAABOHYRYAAAAAMFlE5P8DtgMamIi8PNgAAAAASUVORK5CYII=
```

Comment:
please update the icon, use the new from: https://github.com/kedacore/keda/blob/main/images/logos/keda-word-colour.svg

You can preview the CSV in here: https://operatorhub.io/preview